Assets/General/Scripts/ScreenText/TutorialPopUps.cs

```diff
@@ -35,16 +36,21 @@ public class TutorialPopUps : MonoBehaviour
     [Tooltip("The message shown after completing a tutorial")]
     private const string _CONGRATULATION_MESSAGE = "Great Job!";
 
-    [Tooltip("The diviser to get the message completed in a certain amount of time")]
-    private const float _TOTAL_DISPLAY_TIME = 3f;
+    /*[Tooltip("The diviser to get the message completed in a certain amount of time")]
+    private const float _TOTAL_DISPLAY_TIME = 3f;*/
 
     [Tooltip("The time the final message is displayed")]
     private const float _FINAL_MESSAGE_DISPLAY_TIME = 2f;
 
     // Start's the tutorial process
     void Start()
     {
-        StartCoroutine(TimeBeforeText());
+        if (_uIData.Length > 0 && _uIData != null &&
```

Comment:
maybe wrap this check in a function for readability 
`if(FunctionThatReturnsABool)`